src/assets/styles/styles/custom/_tables.scss

```diff
@@ -166,6 +166,7 @@
       padding-bottom: 50%;
       width: 100%;
       border-left: 1px solid $table-border-color-grey;
+      background-color: $white;
```

Comment:
Nie wiem czy to to, ale jest problem z kolorem tła:
![image](https://user-images.githubusercontent.com/23723122/112290233-4faecd80-8c8f-11eb-9094-1177f983d72f.png)
Tak to wygląda na developie
![image](https://user-images.githubusercontent.com/23723122/112290285-5dfce980-8c8f-11eb-8931-3ca88b22b23b.png)

Reply:
Już jest jak na devie

Reply:
Git